src/latbin/lattice-compose.cc

```diff
@@ -94,6 +102,11 @@ int main(int argc, char *argv[]) {
         std::string key = lattice_reader1.Key();
         KALDI_VLOG(1) << "Processing lattice for key " << key;
         Lattice lat1 = lattice_reader1.Value();
+        // Compute a map from each (t, tid) to (sum_of_acoustic_scores, count)
+        unordered_map<std::pair<int32,int32>, std::pair<BaseFloat, int32>,
+                                            PairHasher<int32> > acoustic_scores;
+        if (!write_compact)
```

Comment:
I don't see why this acoustic-scores-map thing is necessary here because composition with an FST that only has weights on the graph side of the scores, will leave the acoustic scores where they were.